plugin/utils/compiler_builtins.py

```diff
@@ -0,0 +1,197 @@
+"""Get compiler built-in flags."""
+
+import logging as _logging
+
+_log = _logging.getLogger("ECC")
```

Comment:
nit: maybe I lack some knowledge here. Why would we start the name with underscore? This is not a protected class member, right?

Reply:
The underscore notation is actually not only used inside classes but also for members inside modules. Background is, that you make clear - in analogy to a class definition - which parts of your module are belonging to the interface and which are internal and hence shall not be used. This is required due to the way Python works. After doing a

```python
import logging
```

in a file called `my_mod.py`, we can

```python
from my_mod import logging
```

which obviously is nonsense, but works as you would expect.

Long story short: I think we can drop the underscore notation (as this is not a library which is supposed to be used in other apps or libraries) but it's just a bit cleaner. I don't mind too much, so its your call here ;-)

Reply:
Thanks for explaining this. Today I learned smth about python. I am completely convinced.